docs/en/enterprise-edition/content-collections/application-security/get-started/application-security-settings/application-security-settings.adoc

```diff
@@ -28,17 +28,17 @@ Define paths to exclude for scans and specify repositories for targeted applicat
 //xref:code-reviews-pr-comments.adoc[Code Reviews and PR Commits]
 
 
-| Scan Git History for Secrets
+|Validate Secrets
 
 Prima Cloud leverages service provider APIs to verify the authenticity and integrity of secrets discovered within the codebase. This process serves as a crucial step in prioritizing security measures. By confirming the validity of these secrets, you can allocate resources and attention to the most pressing security concerns, ensuring a focused and effective approach to safeguarding sensitive information.
 
-|xref:../../risk-management/monitor-and-manage-code-build/secrets-scanning.adoc[Secrets Scanning]
+|xref:../../risk-management/monitor-and-manage-code-build/secrets-scanning.adoc#validate-secrets[Validate Secrets]
 
-|Validate Secrets
+|Scan Git History for Secrets
 
-Enabling Git History scanning empowers you to uncover any secrets that may have been removed from the repository but remain accessible in its history. This ensures a comprehensive security sweep for vulnerabilities in potentially forgotten or overlooked access points.
+Deleting a secret from a repository does not completely remove its exposure as it can still be found in the git history of the repository. Enabling Git History scanning empowers you to uncover any secrets that may have been removed from the repository but remain accessible in its history. This ensures a comprehensive security sweep for vulnerabilities in potentially forgotten or overlooked access points.
 
-|xref:../../risk-management/monitor-and-manage-code-build/secrets-scanning.adoc#validate-secrets[Validate Secrets]
+|xref:../../risk-management/monitor-and-manage-code-build/secrets-scanning.adoc[Secrets Scanning]
 
```

Comment:
I think that Secrets scanning should be before Validate secrets.

Reply:
It's just a link in a table:
<img width="760" alt="Screenshot 2024-09-26 at 3 58 28 PM" src="https://github.com/user-attachments/assets/7611d5aa-dc04-414c-9a4c-7253c6a278b9">